syntax = "proto3";

message TaskId
{
  int64 value = 1;
}